### PR TITLE
Update texts

### PR DESCRIPTION
**Pull request by JulianRoesner**
# Short Description
We have to update some of the texts in the Steuerlotse for the new tax period. You can find the texts in the [ticket](https://steuerlotse.atlassian.net/browse/STL-1655?atlOrigin=eyJpIjoiMDg2ZWVkOGVmMTU0NGY1YWIzNWVjMTEwOTJlODEzM2EiLCJwIjoiaiJ9)

# Changes
- Update texts
- Add a detail component to the step aussergewöhnliche Belastungen

# Feedback
- What do you think about the way I added the details component? I did not want to adapt the macros because we will probably create a special React page component in the future anyway
- Anything I missed?

# How to review this Pull Request
[Link to Confluence](https://digitalservice4germany.atlassian.net/wiki/spaces/STL/pages/117637157/Development+Process#Approve-or-comment) (internal)

### CODE REVIEW

File: webapp/app/translations/de/LC_MESSAGES/messages.po [hunk 1]
```diff
@@ -1698,522 +1697,568 @@ msgstr "Summe der Rechnungsbeträge"
...
+"Hinweise</strong>"
 
-#: app/forms/steps/lotse/steuerminderungen.py:334
+#: app/forms/steps/lotse/steuerminderungen.py:340
 msgid "form.lotse.steuerminderungen.details-list-item-1"
 msgstr ""
-"Es <strong>muss eine Rechnung</strong> vorliegen, die zum Beispiel per "
-"Überweisung oder EC-Kartenzahlung beglichen worden ist. "
-"<strong>Barzahlungen können nicht</strong> geltend gemacht werden."
+"Es muss eine <strong>Rechnung vorliegen</strong>, die zum Beispiel per "
```
Comment (SannyNguyenHung) [inline webapp/app/translations/de/LC_MESSAGES/messages.po:2085]: Add "Wichtig:" in strong on the beginning

File: webapp/app/translations/de/LC_MESSAGES/messages.po [hunk 2]
```diff
@@ -1698,522 +1697,568 @@ msgstr "Summe der Rechnungsbeträge"
...
+"Barzahlungen können nicht geltend gemacht werden."
 
-#: app/forms/steps/lotse/steuerminderungen.py:335
+#: app/forms/steps/lotse/steuerminderungen.py:341
 msgid "form.lotse.steuerminderungen.details-list-item-2"
 msgstr ""
-"Abgesetzt werden können Kosten, die Sie <strong>im Besteuerungsjahr "
-"bezahlt</strong> haben. Relevant ist der Zeitpunkt der Bezahlung, nicht "
-"der Rechnungsstellung."
+"Abgesetzt werden können Kosten, die Sie im letzen Jahr bezahlt haben. "
```
Comment (SannyNguyenHung) [inline webapp/app/translations/de/LC_MESSAGES/messages.po:2092]:
```suggestion
"Abgesetzt werden können Kosten, die Sie <strong>im letzen Jahr bezahlt haben.</strong>"
```

**Review by SannyNguyenHung — APPROVED** [review]
LGTM! - Just 2 small things :grin:

File: webapp/app/translations/de/LC_MESSAGES/messages.po (hunk 1 above)
Comment (SannyNguyenHung) [reply webapp/app/translations/de/LC_MESSAGES/messages.po:2085]:
```suggestion
"<strong>Wichtig:</strong> Es muss eine <strong>Rechnung vorliegen</strong>, die zum Beispiel per "
```

Comment (frederike-ramin) [reply webapp/app/translations/de/LC_MESSAGES/messages.po:2085]: As I see this now: Are we sure that <strong> is what we want, not <b>?
https://stackoverflow.com/questions/271743/whats-the-difference-between-b-and-strong-i-and-em

**Review by SannyNguyenHung — APPROVED** [review]

**Pull request merged**